Tools/ardupilotwaf/boards.py

```diff
@@ -261,12 +269,26 @@ def configure_env(self, cfg, env):
             'SITL',
         ]
 
+        env.GIT_SUBMODULES += [
+            'uavcan',
+        ]
+
+        if self.with_uavcan:
+            env.INCLUDES += [
+                cfg.srcnode.find_dir('modules/uavcan/libuavcan_drivers/linux/include/').abspath(),
+                cfg.srcnode.find_dir('modules/uavcan/libuavcan/include/').abspath()
+            ]
+
         if sys.platform == 'cygwin':
             env.LIB += [
                 'winmm',
             ]
             env.CXXFLAGS += ['-DCYGWIN_BUILD']
 
+        if self.with_uavcan:
+            env.DEFINES.update(
+                UAVCAN_CPP_VERSION = 'UAVCAN_CPP11'
```

Comment:
maybe just separate this one on another pr. It is should only update the board config